examples/streams/streams.bal

```diff
@@ -23,26 +23,26 @@ function main(string... args) {
     employeeStream.publish(e2);
     employeeStream.publish(e3);
 
-    // Allow for receipt by subscribers. The printEmployeeName function would be invoked for each published event.
+    // Allow for receipt by subscribers. The `printEmployeeName` function is invoked for each published event.
```

Comment:
IMO, would be or should be are more suitable since this is not something that happens here.
What we are trying to say is that during this sleep, the particular function is invoked. 

WDYT?